Source/ChatMgr.cpp

```diff
@@ -748,17 +749,17 @@ namespace GW {
         bool delete_message = false;
         if (sender_encoded) {
             // If message contains link (<a=1>), manually create the message string
-            wchar_t* format = L"\x76b\x10a%s\x1\x10b%s\x1";
+            wchar_t* format = static_cast<wchar_t[]>(L"\x76b\x10a%s\x1\x10b%s\x1");
```

Comment:
The proper fix is `const wchar_t* format`.

Reply:
But then we have to update the following code a lot, since this format variable is used multiple times

Reply:
Yeah, true.

Reply:
VS will error there since non const wchar_t* is expected.
![image](https://user-images.githubusercontent.com/20141069/163232972-d0164898-14c0-40b8-a4ee-60de4553e414.png)

Reply:
So I see one place where it would be an issue (with `time_buffer`), but the pattern used is very bad.

It would make more sense to simply put a local buffer on the stack and memcpy into [here](https://github.com/GregLando113/GWCA/blob/80ef450644f4167337a993a9d25af7abd0364ddd/Source/ChatMgr.cpp#L322), delete the special handle [here](https://github.com/GregLando113/GWCA/blob/80ef450644f4167337a993a9d25af7abd0364ddd/Source/ChatMgr.cpp#L341) and the new [here](https://github.com/GregLando113/GWCA/blob/80ef450644f4167337a993a9d25af7abd0364ddd/Source/ChatMgr.cpp#L326).